packages/convex-helpers/package.json

```diff
@@ -1,96 +1,104 @@
 {
   "name": "convex-helpers",
-  "version": "0.1.46",
+  "version": "0.1.47-alpha.2",
   "description": "A collection of useful code to complement the official convex package.",
   "type": "module",
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
```

Comment:
it looks like you wouldn't be able to `npm install ../../convex-helpers` anymore with this since this package.json now points to files that don't exist

Reply:
You can `npm install ../../convex-helpers/dist` now and it works a little more honestly